filehandling.c

```diff
@@ -0,0 +1,48 @@
+#include<stdio.h>
+
+struct employee
+{
+	char employeeName[50];
+	int employeeId;
+};
+
+int main()
+{
+	FILE *fp=NULL;
+	struct employee emp[2];
+	int i;
+	for(i=0;i<2;i++)
+{
+	printf("Enter the employee details\n");
+	printf("Name:");
+	scanf("%s",emp[i].employeeName);
+	printf("Id:");
+	scanf("%d",&emp[i].employeeId);
+}
+
+fp=fopen("file.bin","wb");
+if(fp==NULL)
+{
+	printf("error in open file in write mode");
+}
+else
+{
+	for(i=0;i<2;i++)
+ fwrite(&emp[i],sizeof(struct employee),1,fp);
+}
+
+fp=fopen("file.bin","rb");
```

Comment:
Always close the file before opening it again.

```suggestion
fclose(fp);

fp=fopen("file.bin","rb");
```

Reply:
I done changes as your review commands . can you please verify once